GDAXSharp/Services/AbstractService.cs

```diff
@@ -127,6 +134,8 @@ public abstract class AbstractService
             string uri,
             string content = null)
         {
+            Log.Debug("REST {HttpMethod} {Uri} {Content}", httpMethod, uri, content);
```

Comment:
why debug and not information? i guess it could be either.....

Reply:
Yep... happy to change it.. I originally had it as Informational - but after running my app for a day and having the log swapped with output from this I changed it to Debug :)